### PR TITLE
update Deal Detail View - Invest

### PR DESCRIPTION
**Pull request by Trading2021**
image
image
image
image
image

### CODE REVIEW

File: frontend/src/Components/DealDetailsView/Components/my-investment.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                        <Th>Amount Invested</Th>
+                        <Th></Th>
+                        </Tr>
+                    </Thead>
+                    <Tbody>
+                        {
+                            props.dealData.nftTokenArray.map((nft) => {
+                                return (
+                                    <Tr>
+                                    <Td px={2}>{nft}</Td>
```
Comment (coinfounded) [inline frontend/src/Components/DealDetailsView/Components/my-investment.js:33]: The NFT name should be {ticker_symbol} #{nft_id}. You can get the ticker symbol using the Moralis NFT API.

File: frontend/src/Components/DealDetailsView/Components/invest.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+function Invest(props) {
+    return (
+        <Container>
+            <VStack py="5" spacing="1">
+                <Select placeholder='Select your NFT'>
+                    {
+                        props.dealData ? props.dealData.nftTokenArray.map((nft, index)=>{
+                            return (
+                                <option value={index}>{nft}</option>
```
Comment (coinfounded) [inline frontend/src/Components/DealDetailsView/Components/invest.js:12]: these options should be loaded in from moralis. moralis lets you get the nfts that the user owns. If the NFT gating is BAYC, we should show all of the BAYC NFTs the user has in their wallet. Here is the contract of BAYC: https://etherscan.io/address/0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d

File: frontend/src/Components/DealDetailsView/Components/step-percent.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { Grid, GridItem, Box, VStack, Heading, Text, Container, Flex, SimpleGrid } from '@chakra-ui/react';
+
+const steps = [
+    { label: "25%", description: "02/05/2022" },
```
Comment (coinfounded) [inline frontend/src/Components/DealDetailsView/Components/step-percent.js:4]: The dates will be UTC timestamps, need to format them

File: frontend/src/Components/DealDetailsView/Components/my-investment.js (hunk 1 above)
Comment (Trading2021) [reply frontend/src/Components/DealDetailsView/Components/my-investment.js]: image
updated

File: frontend/src/Components/DealDetailsView/Components/invest.js (hunk 2 above)
Comment (Trading2021) [reply frontend/src/Components/DealDetailsView/Components/invest.js]: image
updated

File: frontend/src/Components/DealDetailsView/Components/step-percent.js (hunk 3 above)
Comment (Trading2021) [reply frontend/src/Components/DealDetailsView/Components/step-percent.js]: image
updated

**Review by kkarnik — APPROVED** [review]

**Pull request merged**